### PR TITLE
feat: changed Tele Tan length to be configurable

### PR DESCRIPTION
**Pull request by mschulte-tsi**

### CODE REVIEW

File: src/main/java/app/coronawarn/verification/service/TanService.java [hunk 1]
```diff
@@ -249,6 +251,11 @@ protected VerificationTan generateVerificationTan(String tan, TanType tanType, T
     return tanRepository.findByTanHash(hashingService.hash(tan));
   }
 
+  @Value("${tan.tele.valid.length}")
```
Comment (tence) [inline src/main/java/app/coronawarn/verification/service/TanService.java:254]: add default  @Value("${tan.tele.valid.length:9}")

File: src/main/java/app/coronawarn/verification/service/TanService.java [hunk 2]
```diff
@@ -53,7 +54,8 @@
   private static final Pattern TAN_PATTERN = Pattern.compile(TAN_TAN_PATTERN);
 
   // Tele-TANs are a shorter, easier to communicate form of TAN
-  private static final int TELE_TAN_LENGTH = 9;
+  @Value("${tan.tele.valid.length}")
+  private static int TELE_TAN_LENGTH = 9;
```
Comment (jhagestedt) [inline src/main/java/app/coronawarn/verification/service/TanService.java:58]: statics should not (or can not!) be changed with @value
Just use VerificationApplicationConfig here also!

File: src/main/java/app/coronawarn/verification/service/TanService.java (hunk 1 above)
Comment (jhagestedt) [reply src/main/java/app/coronawarn/verification/service/TanService.java:254]: Dont use @Value at all here - just inject VerificationApplicationConfig and use the getters :D Default Value is defined inside VerificationApplicationConfig.

File: src/main/java/app/coronawarn/verification/service/TanService.java (hunk 2 above)
Comment (mschulte-tsi) [reply src/main/java/app/coronawarn/verification/service/TanService.java]: No longer relevant

**Pull request merged**